.github/workflows/buf.yml

```
@@ -3,18 +3,20 @@ name: Buf CI
 on:
   push:
     paths:
+      - .github/workflows/buf.yml
       - proto/**.proto
   pull_request:
     types: [opened, synchronize, reopened, labeled, unlabeled]
     paths:
       - proto/**.proto
   delete:
-permissions:
-  id-token: write
-  contents: read
-  pull-requests: write
 jobs:
   push-module:
+    if: github.repository == 'redpanda-data/console'
```

```suggestion
    if: ${{ github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == 'redpanda-data/console' }}
```
i suggest similar changes to https://github.com/redpanda-data/connect/pull/2825#discussion_r1745848682